doc/deployment_guide.md

```diff
@@ -49,5422 +49,6064 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
-    inet 127.0.0.1/8 scope host lo
-    valid_lft forever preferred_lft forever
-    inet6 ::1/128 scope host
-    valid_lft forever preferred_lft forever
-    19: tap05a22fb4-4f: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc
-    noqueue state UNKNOWN
-    link/ether fa:16:3e:99:b9:88 brd ff:ff:ff:ff:ff:ff
-    inet 192.168.201.2/24 brd 192.168.201.255 scope global tap05a22fb4-4f ->
-    Tenant network
-    valid_lft forever preferred_lft forever
-    inet6 fe80::f816:3eff:fe99:b988/64 scope link
-    valid_lft forever preferred_lft forever
-    ```
-  
-    
-    f.	Ping the IP address of the instance.
-    
-    g.	SSH into the instance, as cirros, using the keypair generated above:
-    
-    ```bash
-    $ ssh -i MY_KEY.pem cirros@<ip>
-    ```
-
-
-
-11. Format the drive and access it.
-
-    a. List storage devices:
-    
-    ```bash
-    $ sudo -i
-    $ fdisk –l
-    ```
-
-
-    b. Format the drive:
-
-    ```bash
-    $ mkfs.ext3 /dev/vdb
-    ```
-
-
-c. Mount the device, access it, and then unmount it:
-
-```bash
-$ mkdir ~/mydrive
-$ mount /dev/vdb ~/mydrive
-$ cd ~/mydrive
-$ touch helloworld.txt
-$ ls
-$ umount ~/mydrive
-```
-
-
-### Scripted RHOSP sanity test
-
-As an alternative to manually testing your deployment script, we provide sanity_test.sh, which tests all of the basic functionality.
-
-**To run the sanity test script:**
-
-1.	Log into the director node using the user name and password specified when creating the node.
-
-2.	Review the pilot/deployment-validation/sanity.ini file, and modify the parameters as appropriate for your environment. If using OVS-DPDK, set the value for  
-      
-    ```bash
-    ovs_dpdk_enabled= to True.
-    ```
-
-	> Note: The sanity test generates the public/private SSH key pair using the name specified in sanity.ini in the sanity_key_name parameter. The public key is named ~/ <sanity_key_name>.pub, and the private key is named ~/<sanity_key_name>.
- 
-
-3.	From your home directory, execute the sanity_test.sh script:
-
-    ```bash
-    $ cd ~/
-    $ ./pilot/deployment-validation/sanity_test.sh
-    ```
-
-4.	If you wish to clean the environment once the sanity_test.sh script has run successfully:
-
-    ```bash
-    $ cd ~/
-    $ ./pilot/deployment-validation/sanity_test.sh clean
-    ```
-
-> Note: There are deployment options [Sanity Test Settings], in the sample files which may be configured initially to run the sanity tests automatically after the overcloud deployment is successful. The relevant sanity logs will be generated in the director VM at the directory of pilot/deployment-validation/
-
-
-
-
-<div style="page-break-after: always;"></div>
-
-# Appendix D Hugepages
-
-This appendix details the guidelines for configuration of Hugepages during the deployment of Dell EMC Ready Architecture for Red Hat OpenStack Platform version 13.0. The number of Hugepages is now calculated at runtime and no longer needs to be specified by the user.
-

```

Comment:
'The number of Hugepages is now calculated at runtime and no longer needs to be specified by the user' -- this line can be removed.